packages/ssz/test/unit/strings.test.ts

```diff
@@ -0,0 +1,59 @@
+import {expect} from "chai";
+
+import {Case} from "../../src/util/strings";
+
+const testCases = ["fooBar", "foo1Bar"];
```

Comment:
Can you add a few more testcases? Maybe "fooBarBaz", "fooBar1Baz".

you can also make tests more generic like:
```ts
const testCases = [
  { input: "fooBar", snake: "foo_bar", pascal: "FooBar" ......},
  ...
]
const cases = ["snake", "pascal", .....]

for (const case of cases) {
  describe(`should convert strings to ${case} case`, () => {
    for (const testCase of testCases) {
      const expected = testCase[case];
      it(`should convert to ${expected}`, () => {
        const field = Case[case](testCase.input);
        expect(field).to.equal(expected);
      });
    }
  })
}
```

Reply:
Done.  I think I got all the typing correct too but let me know if you want a different approach on this.